code/game/machinery/hologram.dm

```diff
@@ -563,11 +541,12 @@ Possible to do for anyone motivated enough:
 			Hologram.add_atom_colour("#77abff", FIXED_COLOUR_PRIORITY)
 			Hologram.Impersonation = user
 
-		Hologram.mouse_opacity = MOUSE_OPACITY_TRANSPARENT//So you can't click on it.
+		//Hologram.mouse_opacity = MOUSE_OPACITY_TRANSPARENT//So you can't click on it. //SKYRAT EDIT -- Customization; Making holograms clickable/examinable
 		Hologram.layer = FLY_LAYER //Above all the other objects/mobs. Or the vast majority of them.
 		SET_PLANE_EXPLICIT(Hologram, ABOVE_GAME_PLANE, src)
 		Hologram.set_anchored(TRUE)//So space wind cannot drag it.
-		Hologram.name = "[user.name] (Hologram)"//If someone decides to right click.
+		// Hologram.name = "[user.name] (Hologram)"//If someone decides to right click. //SKYRAT EDIT -- ORIGINAL
+		Hologram.name = user.name //SKYRAT EDIT -- Make the name exact, so that the double-emotes are less jarring in the chat
```

Comment:
I'm not sure that I understand this change. I think it makes sense that the hologram itself is marked as `(Hologram)`, what should happen is that the person sending the hologram should probably only see the hologram's own emote, and not the one coming from themselves, when they send it over hologram.

Reply:
I did this for simplicity's sake, because doing otherwise is entering spaghetti territory and it's necessary to delve even deeper. Primarily because audible_message() doesn't have an argument where I can pass a list of ignored mobs, so two non-identical, due to different names, messages get doubleposted and clutter the chat.